x-pack/plugins/siem/public/components/page/hosts/authentications_table/index.tsx

```diff
@@ -112,53 +112,35 @@ export const AuthenticationTable = connect(
   }
 )(AuthenticationTableComponent);
 
-const getAuthenticationColumns = (): Array<Columns<AuthenticationsEdges>> => [
+const getAuthenticationColumns = (): [
+  Columns<AuthenticationsEdges>,
+  Columns<AuthenticationsEdges>,
+  Columns<AuthenticationsEdges>,
+  Columns<AuthenticationsEdges>,
+  Columns<AuthenticationsEdges>,
+  Columns<AuthenticationsEdges>,
+  Columns<AuthenticationsEdges>,
+  Columns<AuthenticationsEdges>,
+  Columns<AuthenticationsEdges>
+] => [
   {
     name: i18n.USER,
     truncateText: false,
     hideForMobile: false,
-    render: ({ node }) => {
-      const userName: string | null | undefined = get('user.name[0]', node);
-      if (userName != null) {
-        const id = escapeDataProviderId(`authentications-table-${node._id}-user-${userName}`);
-        return (
-          <DraggableWrapper
-            key={id}
-            dataProvider={{
-              and: [],
-              enabled: true,
-              id,
-              name: userName,
-              excluded: false,
-              kqlQuery: '',
-              queryMatch: {
-                field: 'user.name',
-                value: userName,
-              },
-            }}
-            render={(dataProvider, _, snapshot) =>
-              snapshot.isDragging ? (
-                <DragEffects>
-                  <Provider dataProvider={dataProvider} />
-                </DragEffects>
-              ) : (
-                userName
-              )
-            }
-          />
-        );
-      } else {
-        return getEmptyTagValue();
-      }
-    },
+    render: ({ node }) =>
+      getRowItemDraggables({
+        rowItems: node.user.name,
+        attrName: 'user.name',
+        idPrefix: `authentications-table-${node._id}-userName`,
+      }),
```

Comment:
Happy to see how much `getRowItemDraggables()` cleans up these column declarations -- these incremental improvements are paying off quite nicely.